### PR TITLE
Fixed GeoIP filter

### PR DESCRIPTION
**Pull request by mirdhyn**

### CODE REVIEW

File: filter-geoip/geoip.go [hunk 1]
```diff
@@ -93,13 +95,16 @@ func (p *processor) Configure(ctx veino.ProcessorContext, conf map[string]interf
 
 	p.cache = lrucache.New(p.opt.CacheSize)
 	p.databases = map[string]*geoip2.Reader{}
-	p.load()
-	return p.ConfigureAndValidate(ctx, conf, p.opt)
 
+	if err := p.load(conf["databases"]); err != nil {
+		log.Fatalln(err)
```
Comment (mirdhyn) [inline filter-geoip/geoip.go]: I'm not really happy with using default log... I tried to use p.Logger.Fatalln() but then it is silently ignored :(

**Comment by vjeantet** [comment]
will check it

**Comment by mirdhyn** [comment]
Hmmm... I just realized that I forgot to remove all my debugging println... I will make a new PR, sorry for that.

**Comment by vjeantet** [comment]
don't worry i will do it, i'm pushing some changes to your geoip to match logstash conf, tell me what your think

**Pull request merged**